kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/ManageLessonModals.vue

```diff
@@ -33,8 +33,8 @@
   import { LessonResource } from 'kolibri.resources';
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
   import useSnackbar from 'kolibri.coreVue.composables.useSnackbar';
-  import AssignmentCopyModal from '../../plan/assignments/AssignmentCopyModal';
-  import AssignmentDeleteModal from '../../plan/assignments/AssignmentDeleteModal';
+  import AssignmentCopyModal from '../../common/assignments/AssignmentCopyModal';
```

Comment:
A very logical step is removing the `plan` folder and instead have folders for each main nav item, and redistribute these files to these new folders. But I am planning to keep the `reports` folder to have the Reports Pages that are common to different tabs, but move it to the `/common` folder.